omod/src/main/java/org/openmrs/module/pihcore/fragment/controller/dashboardwidgets/EncountersFragmentController.java

```diff
@@ -86,19 +92,42 @@ else if (patientConfig instanceof String) {
 		}
 		model.put("encounterTypeToUrlMap", encounterTypeToUrlMap);
 
```

Comment:
This looks like there are more changes than there really are as I moved some things around, but this just adds support for an additional (optional) configuration property that will limit the returned encounters to being on or after the latest enrollment date of the given program.  If the patient does not have an active enrollment, then no encounters are returned.  If this configuration is not provided, then this should have no change from before.